administration/package.json

```
@@ -52,7 +52,7 @@
   "scripts": {
     "start": "set -ae; REACT_APP_APPLICATION_COMMIT=$(./application_commit.sh); react-scripts start",
     "build": "set -ae; REACT_APP_APPLICATION_COMMIT=$(./application_commit.sh); react-scripts build",
-    "test": "react-scripts test",
+    "test": "node --experimental-global-webcrypto node_modules/jest/bin/jest.js",
```

Necessary until Node 19. Once we upgrade node to the next LTS version 22 (which has not released yet), can simplify this to just 
```suggestion
    "test": "jest",
```